code/datums/callback.dm

```diff
@@ -59,6 +59,10 @@
 	if(usr)
 		user = WEAKREF(usr)
 
+/datum/callback/Destroy()
+	..()
+	return QDEL_HINT_IWILLGC
```

Comment:
Callbacks should never be getting qdeleted and the hint should be let me live. if there is still a reference to the callback its a valid datum, if the thing its referring to is going away it should just null that var.